Error on node computation: 'ListAttribute' object is not iterable

I'm running Windows 10. I have managed to make this with GUI but I have this issue with command line.
Thank you for the answer.

c:\Users\abszolut\Repos\meshroom>python bin/meshroom_photogrammetry --input E:\pic\cnc_rabbit_symbol --output    E:\pic\cnc_rabbit_symbol\result


[13/13] Publish
Publish
- elapsed time: 0:00:00.062456
[2018-08-01 14:41:12,884][ERROR] Error on node computation: 'ListAttribute' object is not iterable
Traceback (most recent call last):
 File "bin/meshroom_photogrammetry", line 64, in <module>
   meshroom.core.graph.executeGraph(graph, toNodes=toNodes, forceCompute=args.forceCompute, forceStatus=args.forceStatus)
 File "C:\Users\abszolut\Repos\meshroom\meshroom\core\graph.py", line 850, in executeGraph
   chunk.process(forceCompute)
 File "C:\Users\abszolut\Repos\meshroom\meshroom\core\node.py", line 252, in process
   self.node.nodeDesc.processChunk(self)
 File "C:\Users\abszolut\Repos\meshroom\meshroom\nodes\aliceVision\Publish.py", line 48, in processChunk
   outFiles = self.resolvedPaths(chunk.node.inputFiles, chunk.node.output.value)
 File "C:\Users\abszolut\Repos\meshroom\meshroom\nodes\aliceVision\Publish.py", line 35, in resolvedPaths
   for inputFile in inputFiles:
TypeError: 'ListAttribute' object is not iterable

--- a/meshroom/nodes/aliceVision/Publish.py
+++ b/meshroom/nodes/aliceVision/Publish.py
@@ -1,6 +1,6 @@
 from __future__ import print_function
 
-__version__ = "1.0"
+__version__ = "1.1"
 
 from meshroom.core import desc
 import shutil
@@ -48,10 +48,11 @@ class Publish(desc.Node):
         if not chunk.node.output.value:
             return
 
-        outFiles = self.resolvedPaths(chunk.node.inputFiles, chunk.node.output.value)
+        outFiles = self.resolvedPaths(chunk.node.inputFiles.value, chunk.node.output.value)
 
         if not outFiles:
-            raise RuntimeError("Publish: input files listed, but nothing to publish. Listed input files: {}".format(chunk.node.inputFiles))
+            raise RuntimeError("Publish: input files listed, but nothing to publish. "
+                               "Listed input files: {}".format(chunk.node.inputFiles.value))
 
         if not os.path.exists(chunk.node.output.value):
             os.mkdir(chunk.node.output.value)

